hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/CopyToTempViewProcedure.scala

```diff
@@ -34,7 +34,8 @@ class CopyToTempViewProcedure extends BaseProcedure with ProcedureBuilder with L
     ProcedureParameter.optional(4, "end_instance_time", DataTypes.StringType, ""),
```

Comment:
Not relate to this pr, `begin_instance_time` and `end_instance_time` Are these 2 configures names wrong? Should be `begin_instant_time` and `end_instant_time`?

Reply:
other call stack all use begin_instance_time and end_instance_time，seems not wrong